zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -116,9 +116,10 @@ pub fn init_wallet_db<P>(wdb: &WalletDb<P>) -> Result<(), rusqlite::Error> {
             address TEXT NOT NULL,
             value INTEGER NOT NULL,
             memo BLOB,
+            sent_note_type INTEGER,
```

Comment:
```suggestion
            output_pool INTEGER NOT NULL,
```
and place this in between `tx` and `output_index`.

Also update the CHANGELOG to document this table change, and outline that users will need to implement this migration manually (by filling this column based on the `address` column).